cpp/ps0000/ps0000_sol.cpp

```diff
@@ -0,0 +1,8 @@
+#include <iostream> // 2/2
```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`